Libraries/CustomComponents/NavigationExperimental/NavigationCardStack.js

```diff
@@ -221,16 +220,7 @@ class NavigationCardStack extends React.Component<DefaultProps, Props, void> {
   _configureTransition = () => {
     const isVertical = this.props.direction === 'vertical';
     const animationConfig = {};
-    if (
-      !!NativeAnimatedModule
-
-      // Gestures do not work with the current iteration of native animation
-      // driving. When gestures are disabled, we can drive natively.
-      && !this.props.enableGestures
-
-      // Native animation support also depends on the transforms used:
-      && NavigationCardStackStyleInterpolator.canUseNativeDriver(isVertical)
-    ) {
+    if (NavigationCardStackStyleInterpolator.canUseNativeDriver(isVertical)) {
```

Comment:
I added this check because some transforms are(were?) not supported by main thread animations. So depending on the interpolator, we should still avoid native animation.